content/introduction/concepts.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Concepts"
+description: "Learn more about the core concepts and domain model that make up the SweetOps methodology."
+weight: 3
+---
+
+SweetOps is built on top of a number of high-level concepts and terminology that are critical to understand prior to getting started. In this document, we break down these concepts to help you get a leg up prior to going through your first tutorial.
+
+## Layers
+
+As discussed in [Philosophy](TODO), SweetOps breaks down cloud infrastructure into 4 discrete layers:
+
+1. **Foundational**: Your AWS Accounts, VPCs, IAM, and DNS architecture.
+1. **Platform**: Your EKS / ECS Clusters, Load Balancers, IAM Service Accounts, and Certificates.
+1. **Shared Services**: Your CI/CD pipelines, BeyondCorp solution, and Monitoring and Logging tooling.
+1. **Application**: Your Backend and Frontend Applications.
+
+We delineate between these layers as they have different lifecylces and tools that are responsible for managing them (e.g. Terraform is great for building your Foundational layer, but is not a good solution to manage your Application layer). It's also important to note that as you proceed down the layers, each layer is more likely to change over time: You won't frequently change your AWS VPCs in your Foundational layer or rebuild EKS clusters in your Platform layer, but you might add HashiCorp Vault to your Shared Services to increase your security posture or add a new API microservice to your Application layer.
+
+## Components
+
+Components are opionated, self contained units of infrastructure as code that solve one, specific problem. SweetOps has two flavors of components:
```

Comment:
```suggestion
Components are opinionated, self-contained units of infrastructure as code that solve one, specific problem. SweetOps has two flavors of components:
```

Reply:
I write these in VSCode... but I should likely write in GDocs so I get spelling and grammatical suggestions upfront. Will do so going forward.